server/controllers/notification.js

```diff
@@ -66,3 +66,16 @@ exports.getUnreadNotificaiton = asyncHandler(async (req, res, next) => {
       throw new Error(error.message);
     }
 });
+
+exports.setReadNotifications = asyncHandler(async (req, res, next) => {
+  try{
+    const updatedNotifications = await Notification.updateMany(
```

Comment:
The variable 'updateNotifications' has been declared but we are not using it anywhere, I think we can exclude it?